modules/Sources/Models/TransactionState.swift

```diff
@@ -102,18 +102,18 @@ public struct TransactionState: Equatable, Identifiable {
         }
     }
 
-    public var roundedAmountString: String {
-        let formatted = zecAmount.decimalZashiFormatted()
-        
-        switch status {
-        case .paid, .sending:
-            return "-\(formatted)"
-        case .received, .receiving:
-            return "+\(formatted)"
-        case .failed:
-            return isSentTransaction ? "-\(formatted)" : "+\(formatted)"
-        }
-    }
+//    public var roundedAmountString: String {
```

Comment:
Should we remove this instead of commenting it out?

Reply:
good catch, removed